homeassistant/components/homekit_controller/climate.py

```diff
@@ -200,7 +237,7 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
                 {CharacteristicsTypes.ACTIVE: ActivationStateValues.INACTIVE}
             )
             return
-        if hvac_mode not in {HVACMode.HEAT, HVACMode.COOL}:
+        if hvac_mode not in {HVACMode.HEAT, HVACMode.COOL, HVACMode.HEAT_COOL}:
```

Comment:
The only values we can ever support are these 3, so  the _LOGGER.warning and this whole if clause can go away.